File: data/CodeLlama/readable_format_HPEC/cpp/file112.cpp
#include <iostream>
#include <omp.h>

int main() {
    int k;

    #pragma omp parallel
    {
        #pragma omp master
        {
            k = omp_get_num_threads();
            std::cout << "Number of threads requested = " << k << std::endl;
        }
    }

    return 0;
}

File: data/StarCoder2/readable_format_HPEC/fortran/file112.f90
program DRB103_master_orig_no
    use omp_lib
    implicit none

    integer k

    !$omp parallel
    !$omp master
    k = omp_get_num_threads()
    print 100, k
    100 format ('Number of threads requested =',3i8)
    !$omp end master
    !$omp end parallel

end program
